mathesar_ui/src/components/select/Select.svelte

```diff
@@ -45,7 +45,9 @@
   }
 
   function setSelectedItem() {
-    currentIndex = options.indexOf(value);
+    if (options.some((e) => e.label === value[labelKey])) {
```

Comment:
The idKey exists for the sole purpose of acting as the identifier for an option, in cases of checking conditions as such.

We can change this to:
```suggestion
    if (options.some((e) => e[idKey] === value?.[idKey])) {
```